src/Core/Auth/Repositories/IAuthRequestRepository.cs

```diff
@@ -5,7 +5,7 @@ namespace Bit.Core.Repositories;
 
 public interface IAuthRequestRepository : IRepository<AuthRequest, Guid>
 {
-    Task<int> DeleteExpiredAsync();
+    Task<int> DeleteExpiredAsync(TimeSpan userExpiration, TimeSpan adminExpiration, TimeSpan adminApprovalExpiration);
```

Comment:
Minor (non-blocking) ❓ :  why the shortened names for the params here vs the full names? 

I found the full names to be very clear, but you lose a decent bit of clarity w/ the shorter names.
Example: `userExpiration` !== `UserRequestExpiration` at a glance

Reply:
Not sure why I dropped the `Request`. Added it back!